### PR TITLE
Classic gray theme: Products view

### PR DESCRIPTION
**Pull request by tulimaki**
* Add products view to classic gray theme
* Show 9 newest products in products view
* Add link to products view in main navigation
* Drop categories from main navigation
* Make main navigation current page work
* Add products view link to category views breadcrumbs

Refs SHOOP-1402

### CODE REVIEW

File: shoop/themes/classic_gray/templates/classic_gray/products_view.jinja [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-3 categories-nav">
+            {% include "shoop/front/includes/_category_menu.jinja" %}
+        </div>
+        <div class="col-md-9">
+            <div class="row product-list-actions">
+                <div class="toggle-view">
+                    <span class="control-label">Show Grid/List</span>
```
Comment (Pikkupomo) [inline shoop/themes/classic_gray/templates/classic_gray/products_view.jinja]: missing {% trans %}

File: shoop/themes/classic_gray/templates/classic_gray/products_view.jinja [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                    <input id="product-list-view-type" type="checkbox">
+                    <label data-original-title="Grid / list view" for="product-list-view-type" data-toggle="tooltip" title="">
```
Comment (Pikkupomo) [inline shoop/themes/classic_gray/templates/classic_gray/products_view.jinja]: title missing {% trans %}

File: shoop/themes/classic_gray/templates/classic_gray/shoop/front/includes/_navigation.jinja [hunk 3]
```diff
@@ -113,26 +113,15 @@
...
-                    <li>
-                        {% if children %}
-                            <a href="{{ url("shoop:category", pk=c.pk, slug=c.slug) }}" class="dropdown-toggle" data-toggle="dropdown">{{ c }} <span class="caret"></span></a>
-                            <ul class="dropdown-menu">{{ loop(children) }}</ul>
-                        {% else %}
-                            <a href="{{ url("shoop:category", pk=c.pk, slug=c.slug) }}">{{ c }}</a>
-                        {% endif %}
-                    </li>
-                {% endfor %}
+                <li{% if request.path == "/" %} class="current"{% endif %}><a href="/">Home</a></li>
```
Comment (Pikkupomo) [inline shoop/themes/classic_gray/templates/classic_gray/shoop/front/includes/_navigation.jinja]: missing {% trans %} on Home

**Comment by Pikkupomo** [comment]
Rebase please

**Comment by akx** [comment]
This is probably related to #136 too.

File: shoop/themes/classic_gray/templates/classic_gray/products_view.jinja [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-3 categories-nav">
+            {% include "shoop/front/includes/_category_menu.jinja" %}
+        </div>
+        <div class="col-md-9">
+            <div class="row product-list-actions">
+                <div class="toggle-view">
+                    <span class="control-label">{% trans %}Show Grid/List{% endtrans %}</span>
```
Comment (Pikkupomo) [inline shoop/themes/classic_gray/templates/classic_gray/products_view.jinja]: `Show Grid / List`

Comment (tulimaki) [inline shoop/themes/classic_gray/templates/classic_gray/products_view.jinja]: Well it was originally "Shop Grid/List" so I am not sure do we need those extra spaces?

**Pull request merged**